signac/__main__.py

```diff
@@ -568,20 +568,17 @@ def _sig(st):
         diff_src = error.schema_src.difference(error.schema_dst)
         diff_dst = error.schema_dst.difference(error.schema_src)
         only_in_dst = diff_dst.difference(diff_src)
-        only_in_src = diff_src.difference(diff_src)
+        only_in_src = diff_src.difference(diff_dst)
         diff_value = diff_src.intersection(diff_dst)
         if only_in_src:
-            keys_formatted = ('.'.join(k) for k in only_in_src)
```

Comment:
@atravitz why did you remove this? Are nested statepoints being converted from a list representation to a dot-separated string upstream of this?

Reply:
I confirmed that the keys are converted to a dot-separated string. This is safe to remove.